bundles/org.openhab.binding.miio/src/test/java/org/openhab/binding/miio/internal/ReadmeHelper.java

```diff
@@ -79,8 +79,14 @@ public static void main(String[] args) {
     }
 
     private StringWriter deviceList() {
+        long items = Arrays.asList(MiIoDevices.values()).stream()
+                .filter(device -> !device.getThingType().equals(MiIoBindingConstants.THING_TYPE_UNSUPPORTED)).count();
+        String devicesCount = String.format("Currently the miio binding supports more than %.0f different models.",
+                Math.floor(items / 10) * 10);
```

Comment:
Why format it as a floating point? Why not just use `%d` instead so it prints as an integer?
```suggestion
        String devicesCount = String.format("Currently the miio binding supports more than %d different models.",
                items;
```

Reply:
I had that indeed but it gave me an error as it considered the outcome of the calc a double

Reply:
Ah, I just realized that you are trying to round to the nearest 10. In that case, since `items` is a `long`, all you would need to do is `(items/10)*10`.
Math.floor returns a double.

Reply:
I want it to round down to the lower  10. I guess that I could deduct 5 and keep it as long, but I think the floor function is cleaner

Reply:
what you think is cleaner to round down to the nearest 10?
`((items-5)/10)*10 `  and %d    or  `Math.floor(items / 10) * 10);` and %0.f  both would show the same. My idea was the latter but I can't be too bothered about it

Reply:
Dividing an integer by another integer will automatically floor the result. But if you are more comfortable with using Math.floor and `%0.f` then I'm not going to stop you.

Reply:
ah... got it... thought that way it was doing a regular round instead of a floor.